### ISSUE TITLE
Generated waves on people ID as Global Condition?

### ISSUE THREAD

**Issue by weixsong** [ibab/tensorflow-wavenet#202]
Hi, Does anyone has tried the latest code that integrated the global condition?
If anyone has generated some good waves, could you please share it to me?
I'm interested in what the wave with people ID as global condition sounds like, I'm still training the model, but could not got some any good wave. maybe too less mini-batch now.

**Comment by akademi4eg** [comment]
Here are two examples generated by network after 100k iterations. The code was from @jyegerlehner 's branch that has been merged with latest PR. Also I lowered the learning rate and silence threshold twice during training.

https://soundcloud.com/dmytro-tkanov/gc-wavenet-speaker311-100k-iterations
https://soundcloud.com/dmytro-tkanov/gc-wavenet-225-100k

**Comment by weixsong** [comment]
@akademi4eg , thanks very much.
